### DISCUSSION TITLE
List of assigned issues and PRs are empty

### DISCUSSION THREAD

**Discussion by Linuus** [magit/forge#469] [Support] [answered]
Hi!

I'm using Doom Emacs and I'm trying to add the assigned PRs and issues sections in the magit status buffer but it doesn't work.

I run this:
```elisp
(after! magit
  (magit-add-section-hook 'magit-status-sections-hook 'forge-insert-assigned-pullreqs nil t)
  (magit-add-section-hook 'magit-status-sections-hook 'forge-insert-assigned-issues nil t)
  (magit-add-section-hook 'magit-status-sections-hook 'forge-insert-requested-reviews nil t))
```

But the new sections doesn't show up.

When I'm running: `(forge--ls-assigned-issues (forge-get-repository nil))` it returns `nil`. I do have assigned PRs and issues in this repo.

Any ideas what might be wrong?

**Comment by tarsius** [comment]
With what forge/host do you have this issue? If it's not github.com, then did you try reproducing the issue with that?

**Reply by Linuus** [reply]
It’s GitHub.com

**Reply by tarsius** [reply]
Check if `(ghub--username (forge-get-repository t))` returns the expected value.

Another possible cause is that Github does not consider all modifications to actually count as modifications. Forge only updates modified topics and if Github lies about it, then that's that. I don't know if "add assignee" is one of the modifications that don't count. You can (re-)fetch individual topics (using `n f t`) and if that fixes it, then this probably is the issue (and we cannot do anything about it).

**Reply by Linuus** [reply]
Hm, could it be a casing issue perhaps? It returns `linuus` but it shows here as "Linuus" 🤔 
I wonder what issues I could get into if I change it to lowercase... Shouldn't be too bad.

But still, I guess it should work either way.

**Reply by tarsius** [reply]
That must be the issue.

**Reply by Linuus** [reply]
It was! I changed my authinfo file and my gitconfig to use the "correct" username and now it works!
Thanks!

**Comment by Linuus** [comment, marked as answer]
Solution was to change gitconfig etc. to use the correct casing in my username.
